3_managing_roles/README.md

```diff
@@ -1,3 +1,108 @@
 # Managing Roles
 
-A collection of examples regarding managing roles.
+A collection of examples regarding managing Roles.
+
+## Migrating to Access Rules
+
+To increase flexibility when managing thousands of Resources, Role Grants have
+been deprecated in favor of Access Rules, which allow you to grant access based
+on Resource Tags and Type. Role Grants will be removed from the Terraform
+provider by March 15, 2022.
+
+The following examples demonstrate the deprecated Role Grants, Dynamic Access
+Rules with Tags and Resource Types, and Static Access Rules for backwards
+compatibility with Role Grants.
+
+### Role Grants (deprecated)
+
+Previously, you would grant a Role access to specific resources by ID via Role
+Grants:
+
+```hcl
```

Comment:
`hcl` is more generic than `tf` or `terraform` - is there a reason for that here, do you know? Local previews suggest that our syntax highlighter takes all three.

Reply:
Oh, `hcl` is what was in the migration notes from Product, and I left in in there. I can change to `tf`.